tests/collections/asr/test_audio_modules.py

```diff
@@ -414,3 +415,39 @@ def test_flex_channels(
                         assert torch.all(
                             mask_length == spec_length
                         ), f'Output length mismatch: expected {spec_length}, got {mask_length}'
+
+    @pytest.mark.unit
+    def test_gss(self):
+        """Test initialization of the GSS mask estimator and make sure it can process an input tensor.
+        This tests initialization and the output shape. It does not test correctness of the output.
+        """
+        # Input dimensions
+        num_channels_tests = [1, 4]
+        num_subbands_tests = [32, 65]
+        num_outputs_tests = [2, 3]
+        batch_size = 4
+        num_frames = 50
+
+        # Instantiate UUT
+        uut = MaskEstimatorGSS()
+
+        # Process different channel configurations
+        for num_channels in num_channels_tests:
+            for num_subbands in num_subbands_tests:
+                for num_outputs in num_outputs_tests:
```

Comment:
I think these for loops need to be removed than `num_channels`, `num_subbands`, and `num_outputs` need to be parameterized using `@pytest.mark.parametrize()` since this is the way the test runs can be parallelized during CI test.

Both `test_gss` and `test_flex_channels` can be benefited from this if you make the change.

Reply:
Understood. Probably overhead of parameterizing is surpassing the gain from parameterization.
If this is the case, it should go with the current form.